runtime.md

```diff
@@ -33,24 +33,79 @@ This is provided so that consumers can find the container's configuration and ro
 }
 ```
 
-## Lifecycle
+## Operations
+
+OCI compliant runtimes MUST support the following operations, unless support for the operation can not be supported by the base operating system (kernel) itself.
+
+OPEN: we should consider specifying the options/args for each operation.
+For example, checkpoint (for runc) seems to have a lot of params.
 
 ### Create
 
-Creates the container: file system, namespaces, cgroups, capabilities.
+Using the `config.json` and `runtime.json` files, along with the file system path referenced in the `config.json`, this operation MUST create a new container.
+This includes creating the relevant namespaces, cgroups and configuring the appropriate capabilities for the container.
+
+### Start
+
+This operation MUST start a previously created container.
+Starting a container MUST create a new process, as specified by the `config.json` file within the scope of the container.
+Attempting to start an already running container MUST have no effect on the container and MUST generate an error.
+Starting a stopped container, one that had been previously be running, MUST create a new process, as specified by the `config.json` file, within the scope of the container.
```

Comment:
On Mon, Jan 04, 2016 at 10:47:19AM -0800, Doug Davis wrote:
> >  
> >  ### Create
> >  
> > -Creates the container: file system, namespaces, cgroups, capabilities.
> > +Using the `config.json` and `runtime.json` files, along with the file system path referenced in the `config.json`, this operation MUST create a new container.
> > +This includes creating the relevant namespaces, cgroups and configuring the appropriate capabilities for the container.
> > +
> > +### Start
> > +
> > +This operation MUST start a previously created container.
> > +Starting a container MUST create a new process, as specified by the `config.json` file within the scope of the container.
> > +Attempting to start an already running container MUST have no effect on the container and MUST generate an error.
> > +Starting a stopped container, one that had been previously be running, MUST create a new process, as specified by the `config.json` file, within the scope of the container.
> 
> What specific wording change are you suggesting here?

That the “Create” and “Start” sections are rolled into a single
“Start” entry until we have more clarity on how separate steps would
work.  For my take at unified start/create wording, see [1].  Although
I also point out a number of open issues there (which I suggest we
don't land in this repository until we've sorted them out [2]).

[1]: https://groups.google.com/a/opencontainers.org/d/msg/dev/frUXLljXy8Y/aKFRX56qAAAJ
     Subject: Re: Documenting container lifecycles
     Date: Sat, 19 Sep 2015 14:01:06 -0700
     Message-ID: <20150919210106.GA30120@odin.tremily.us>
[2]: https://github.com/opencontainers/specs/pull/225/files#r48766534